arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -740,7 +741,7 @@ public void tracePostExecution(MessageFrame frame, Operation.OperationResult ope
      */
     if (isExceptional()) {
       this.currentTraceSection()
-          .addFragments(ContextFragment.executionProvidesEmptyReturnData(this));
+          .exceptionalContextFragment(ContextFragment.executionProvidesEmptyReturnData(this));
```

Comment:
Subtle change: previously we were adding the exceptional context row during `tracePostExecution`. The issue was with sections that defer their tracing to a later point in time. We did find ourselves adding "undoing" account or storage fragments at e.g. `tracePostRollback` _after_ having added the exceptional context row.

The new approach is thus to simply _log_ that exceptional context row and add it later on when everything has been done in terms of adding rows. Currently that is: at line counting time.